src/data/home/home.json

```diff
@@ -170,6 +170,39 @@
       "value": "Bitte schreiben Sie uns eine Nachricht und wir melden uns zurück."
     }
   ],
+  "team": {
+    "header": [
+      {
+        "locale": "en",
+        "value": "Team"
+      },
+      {
+        "locale": "de",
+        "value": "Team"
+      }
+    ],
+    "subheader": [
+      {
+        "locale": "en",
+        "value": "The team"
+      },
+      {
+        "locale": "de",
+        "value": "Das Team"
+      }
+    ],
+    "footer": [
+      {
+        "locale": "en",
+        "value": "That is our team"
+      },
+      {
+        "locale": "de",
+        "value": "Das ist unser Team"
+      }
+    ],
+    "members": []
+  },
```

Comment:
Looks like this is suddenly required. Not a problem though because all this stuff wont be rendered on the landingpage as long as we do not add team members.